app/components/GLVolmeters.tsx

```diff
@@ -108,7 +108,7 @@ export default class GLVolmeters extends TsxComponent<VolmetersProps> {
   private sourcesOrder: string[];
   private workerId: number;
   private requestedFrameId: number;
-  private bgMultiplier = this.customizationService.isDarkTheme ? 0.2 : 0.5;
+  private bgMultiplier = this!.customizationService.isDarkTheme ? 0.2 : 0.5;
```

Comment:
kind of wondering why we need to assert `this`, but only sometimes?

Reply:
Typescript thinks that `this!.customizationService` is not initialized here.